### ISSUE TITLE
Typescript breakpoints do not work when outfiles contain a glob pattern

### ISSUE THREAD

**Issue by kpdonn** [Microsoft/vscode#45377]
- VSCode Version: Version 1.21.0
- OS Version: macOS High Sierra 10.13.2

Steps to Reproduce:

1. Have a launch.json option like the following, where the outfiles is a glob pattern for js files generated by typescript:
```
    {
      "type": "node",
      "request": "attach",
      "name": "Attach",
      "port": 9229,
      "sourceMaps": true,
      "outFiles": [
        "${workspaceFolder}/built/local/*.js"
      ]
    }
```
2. Debug a typescript file with node's `--inspect-brk` option.
3. Attach to the node process for debugging using a configuration like the one above.
4. Set a breakpoint in the typescript file
5. Continue so that the breakpoint should be hit but notice it never actually stops.

Then repeat the same process but specifying the generated js file directly instead of using a glob pattern and notice the breakpoint is hit as expected in that case.

I doubt it would affect anything but if there are issues reproducing it, the project I experienced this with was the main TypeScript repository, using their [instructions](https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#debugging-the-tests) for debugging the typescript tests.

Does this issue occur when all extensions are disabled?: Yes

Potentially related or the same issue:
* #45351 
* #45243
* https://github.com/Microsoft/vscode-react-native/issues/625

**Comment by roblourens** [comment]
Should it be `"${workspaceFolder}/built/local/**/*.js"` (all children of local, not just direct children)?

**Comment by kpdonn** [comment]
@roblourens So this is strange but today I no longer can reproduce this problem at all. 

When I made this ticket I had reproduced it several times in different ways including using your suggested `${workspaceFolder}/built/local/**/*.js` pattern and consistently noticed that when it contained any sort of `*` or `**` wildcards it wouldn't hit the breakpoints, but when I directly listed the correct file without wildcards it would hit the breakpoints. I did it several times in a row going back and forth between with and without wildcards just to confirm to myself that that was definitely the issue.

Today though it works with any combination of with or without the wildcards. I'm at a loss to explain what is different. I did rebuild the project, but not in any different way than I had built it before. 

Since I'm unable to reproduce this myself anymore I'm going to close this. I am new to VS Code so it's possible this was some form of user error, although I was concerned about that before I created the issue and only created it after I was confident it wasn't. I'll reopen if I'm able to reproduce it again.
